pkg/maps/nat/nat.go

```diff
@@ -331,6 +333,7 @@ func DeleteMapping6(m *Map, ctKey *tuple.TupleKey6Global) error {
 
 		m.SilentDelete(&key)
 		m.SilentDelete(&rkey)
+		metrics.NatDeleteMappingv6Events.Inc()
```

Comment:
If this is in the context of GC observability, should we also increment this in the DeleteSwappedMapping{4,6} case?

If so, maybe this is better off handled in ctmap code (i.e. such as in the purgeCtEntry{4,6} funcs `pkg/maps/ctmap/ctmap.go`

Reply:
I will check ctmap.go. Do we not need to increment the counters here in deletemapping?

Reply:
What I was thinking is that because this metric is related to GC, not just map deletes, we would want to do this at the GC logic and (potentially) provide some context as to what kind of eviction event it is. 

From what I can tell,  the NAT DeleteMapping functions are only used by ctmap GC - so they're currently one in the same. However it is possible that some new use case could arise for the functions that isn't necessarily related to GC which might pollute the data.